src/mudlet.cpp

```diff
@@ -500,8 +500,45 @@ mudlet::mudlet()
     initEdbee();
 }
 
+QPointer<QSettings> mudlet::getQConfig()
+{
+    QPointer<QSettings> config;
+    // Try for a mudlet config file, first, and check whether a custom
+    // path for the configuration is given
+    config = new QSettings(QCoreApplication::applicationDirPath() + "/config.ini", QSettings::IniFormat);
```

Comment:
This is NOT the past/currently used main Mudlet `QSettings` file which (on a Linux PC at least, *now*) is `~/.config/mudlet/Mudlet.conf` {the capitalisation is forced on it because the application name is "Mudlet"} - should you not be trying there first?

Reply:
That comment is actually a bit misworded. That function only looks for the `config.ini` file that contains `configPath`. I left `getQSettings()` to look for the Mudlet configuration file.